libEGL warning: DRI3 error: Could not get DRI3 device
libEGL warning: Activate DRI3 at Xorg or build mesa with DRI2
libEGL warning: DRI3 error: Could not get DRI3 device
libEGL warning: Activate DRI3 at Xorg or build mesa with DRI2
WARNING: All log messages before absl::InitializeLog() is called are written to STDERR
I0000 00:00:1748284530.251699   15167 gl_context_egl.cc:85] Successfully initialized EGL. Major : 1 Minor: 5
I0000 00:00:1748284530.510376   15354 gl_context.cc:369] GL version: 3.2 (OpenGL ES 3.2 Mesa 25.1.1), renderer: llvmpipe (LLVM 19.1.7, 256 bits)
INFO: Created TensorFlow Lite XNNPACK delegate for CPU.
W0000 00:00:1748284530.613237   15297 inference_feedback_manager.cc:114] Feedback manager requires a model with a single signature inference. Disabling support for feedback tensors.
W0000 00:00:1748284530.654393   15302 inference_feedback_manager.cc:114] Feedback manager requires a model with a single signature inference. Disabling support for feedback tensors.
INFO:     Started server process [15167]
INFO:     Waiting for application startup.
INFO:     Application startup complete.
INFO:     Uvicorn running on http://127.0.0.1:8003 (Press CTRL+C to quit)
*** Aborted at 1748284635 (unix time) try "date -d @1748284635" if you are using GNU date ***
PC: @     0x7fd0c8b97ee0 epoll_pwait
*** SIGTERM (@0x3e800003acc) received by PID 15167 (TID 0x7fd0c8a69740) from PID 15052; stack trace: ***
    @     0x7fd0c8b0fed3 (/usr/lib/x86_64-linux-gnu/libc.so.6+0xa1ed2)
    @     0x7fcfbd27b138 (/home/znasif/anaconda3/envs/whatsai2/lib/python3.10/site-packages/pycolmap/_core.cpython-310-x86_64-linux-gnu.so+0x69b137)
    @     0x7fd0c8ab3330 (/usr/lib/x86_64-linux-gnu/libc.so.6+0x4532f)
    @     0x7fd0c8b97ee0 epoll_pwait
    @     0x7fd0c74cfd7c uv__io_poll
    @     0x7fd0c74be40a uv_run
    @     0x7fd0c75a8c11 __pyx_f_6uvloop_4loop_4Loop__Loop__run
    @     0x7fd0c75daeaf __pyx_f_6uvloop_4loop_4Loop__run
    @     0x7fd0c75d2ea9 __pyx_pw_6uvloop_4loop_4Loop_25run_forever
    @     0x7fd0c75d4f36 __pyx_pw_6uvloop_4loop_4Loop_45run_until_complete
    @     0x55e57e41519d _PyEval_EvalFrameDefault
    @     0x55e57e424b7c _PyFunction_Vectorcall
    @     0x55e57e419457 _PyEval_EvalFrameDefault
    @     0x55e57e424b7c _PyFunction_Vectorcall
    @     0x55e57e41519d _PyEval_EvalFrameDefault
    @     0x55e57e424b7c _PyFunction_Vectorcall
    @     0x55e57e43094c PyObject_Call
    @     0x55e57e4176de _PyEval_EvalFrameDefault
    @     0x55e57e424b7c _PyFunction_Vectorcall
    @     0x55e57e43094c PyObject_Call
    @     0x55e57e4176de _PyEval_EvalFrameDefault
    @     0x55e57e42ffb2 method_vectorcall
    @     0x55e57e43094c PyObject_Call
    @     0x55e57e4176de _PyEval_EvalFrameDefault
    @     0x55e57e424b7c _PyFunction_Vectorcall
    @     0x55e57e41519d _PyEval_EvalFrameDefault
    @     0x55e57e430234 method_vectorcall
    @     0x55e57e4176de _PyEval_EvalFrameDefault
    @     0x55e57e41ce90 _PyObject_FastCallDictTstate.localalias
    @     0x55e57e42e5b9 _PyObject_Call_Prepend
    @     0x55e57e4eef29 slot_tp_call
    @     0x55e57e41da03 _PyObject_MakeTpCall.localalias
Traceback (most recent call last):
  File "/home/znasif/anaconda3/envs/whatsai2/bin/uvicorn", line 10, in <module>
    sys.exit(main())
  File "/home/znasif/anaconda3/envs/whatsai2/lib/python3.10/site-packages/click/core.py", line 1161, in __call__
    return self.main(*args, **kwargs)
  File "/home/znasif/anaconda3/envs/whatsai2/lib/python3.10/site-packages/click/core.py", line 1082, in main
    rv = self.invoke(ctx)
  File "/home/znasif/anaconda3/envs/whatsai2/lib/python3.10/site-packages/click/core.py", line 1443, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/znasif/anaconda3/envs/whatsai2/lib/python3.10/site-packages/click/core.py", line 788, in invoke
    return __callback(*args, **kwargs)
  File "/home/znasif/anaconda3/envs/whatsai2/lib/python3.10/site-packages/uvicorn/main.py", line 412, in main
    run(
  File "/home/znasif/anaconda3/envs/whatsai2/lib/python3.10/site-packages/uvicorn/main.py", line 579, in run
    server.run()
  File "/home/znasif/anaconda3/envs/whatsai2/lib/python3.10/site-packages/uvicorn/server.py", line 66, in run
    return asyncio.run(self.serve(sockets=sockets))
  File "/home/znasif/anaconda3/envs/whatsai2/lib/python3.10/asyncio/runners.py", line 44, in run
    return loop.run_until_complete(main)
  File "uvloop/loop.pyx", line 1518, in uvloop.loop.Loop.run_until_complete
  File "/home/znasif/anaconda3/envs/whatsai2/lib/python3.10/site-packages/uvicorn/server.py", line 70, in serve
    await self._serve(sockets)
  File "/home/znasif/anaconda3/envs/whatsai2/lib/python3.10/site-packages/uvicorn/server.py", line 77, in _serve
    config.load()
  File "/home/znasif/anaconda3/envs/whatsai2/lib/python3.10/site-packages/uvicorn/config.py", line 435, in load
    self.loaded_app = import_from_string(self.app)
  File "/home/znasif/anaconda3/envs/whatsai2/lib/python3.10/site-packages/uvicorn/importer.py", line 19, in import_from_string
    module = importlib.import_module(module_str)
  File "/home/znasif/anaconda3/envs/whatsai2/lib/python3.10/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1006, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 688, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/znasif/vidServer/server/processors/camio_processor.py", line 930, in <module>
    processor = MapIOProcessor()
TypeError: Can't instantiate abstract class MapIOProcessor with abstract method process_pointcloud
